ingestion/src/metadata/ingestion/source/glue.py

```diff
@@ -199,14 +238,14 @@ def get_downstream_tasks(self, task_unique_id, tasks):
                             list(self.task_id_mapping.values()).index(
                                 edges["DestinationId"]
                             )
-                        ][:63]
+                        ][:64]
                     )
         return downstreamTasks
 
     def get_tasks(self, tasks):
         taskList = []
         for task in tasks["Graph"]["Nodes"]:
-            task_name = task["Name"][:63]
+            task_name = task["Name"][:64]
```

Comment:
taskname increase task.name to 128 as well in schema and here

Reply:
Ok